src/machi.proto

```diff
@@ -258,8 +272,9 @@ message Mpb_Request {
     optional Mpb_AppendChunkReq append_chunk = 112;
     optional Mpb_WriteChunkReq write_chunk = 113;
     optional Mpb_ReadChunkReq read_chunk = 114;
-    optional Mpb_ChecksumListReq checksum_list = 115;
-    optional Mpb_ListFilesReq list_files = 116;
+    optional Mpb_TrimChunkReq trim_chunk = 115;
```

Comment:
In theory, these code assignments should be static.  But we don't have to worry about PB backward compatibility yet, so I'm not worried.